cmd/satellite/main.go

```diff
@@ -74,18 +83,26 @@ var (
 	diagCfg struct {
 		BasePath string `default:"$CONFDIR" help:"base path for setup"`
 	}
+	qdiagCfg struct {
+		BasePath  string        `default:"$CONFDIR" help:"base path for setup"`
+		MaxRepair int           `help:"maximum segments that can be repaired concurrently" default:"100"`
+		Interval  time.Duration `help:"how frequently checker should audit segments" default:"3600s"`
+	}
 
-	defaultConfDir = "$HOME/.storj/satellite"
-	defaultDiagDir = "postgres://postgres@localhost/pointerdb?sslmode=disable"
+	defaultConfDir  = "$HOME/.storj/satellite"
+	defaultDiagDir  = "postgres://postgres@localhost/pointerdb?sslmode=disable"
+	defaultQAddrDir = "redis://127.0.0.1:6379?db=1&password=abc123"
```

Comment:
I think the default port is 6378

Reply:
I ran a redis server locally it was running on 6379. so I took it as default. I did google search it says 6379. Also using the command line you can change the location and port for redis server

Reply:
Yes, sure but for some reason our default for repair queue is 6378 https://github.com/storj/storj/blob/master/pkg/datarepair/repairer/config.go#L25

Reply:
Done